launch/view_hlds_laser.launch.py

```diff
@@ -0,0 +1,39 @@
+# /*******************************************************************************
```

Comment:
You used documentation comments for C++. Please change to Python.

Reply:
My fault. I've update launch file.